packages/gatsby-adapter-netlify/src/index.ts

```diff
@@ -17,12 +18,17 @@ async function getCacheUtils(): Promise<undefined | INetlifyCacheUtils> {
   if (_cacheUtils) {
     return _cacheUtils
   }
-  if (process.env.NETLIFY) {
-    const CACHE_DIR = `/opt/build/cache`
+  let CACHE_DIR: string | undefined
+  if (process.env.NETLIFY_LOCAL) {
+    CACHE_DIR = join(process.cwd(), `.netlify`, `build-cache`)
```

Comment:
actually storing and restoring `.cache` and `public` dirs when running `ntl` locally to restore `public` as-is (before adapter potentially moved files around for path prefix and/or no trailing slash purposes)